observability/src/main/java/io/grpc/observability/Observability.java

```diff
@@ -17,15 +17,31 @@
 package io.grpc.observability;
 
 import io.grpc.ExperimentalApi;
+import io.grpc.observability.interceptors.LoggingChannelInterceptor;
 
-/**
- * The main class for gRPC Observability features.
- */
+/** The main class for gRPC Observability features. */
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/8869")
 public final class Observability {
+  private static boolean initialized = false;
+
+  /** Initialize grpc-observability. */
+  public static synchronized void grpcInit() {
+    if (initialized) {
+      throw new IllegalStateException("Observability already initialized!");
+    }
+    LoggingChannelProvider.init(new LoggingChannelInterceptor.FactoryImpl());
```

Comment:
How about gracefully handling `ProviderNotFoundException` that is possibly thrown by `ManagedChannelProvider.provider()`? Or document that `grpcInit` may throw `ProviderNotFoundException`, in which case `grpcFinish()` need/should not be called (otherwise it will throw `IllegalStateException `).

Reply:
added "throws" javadoc